src/Loop.php

```diff
@@ -210,6 +198,21 @@ public static function unreference($eventIdentifier)
         self::get()->unreference($eventIdentifier);
     }
 
+    /**                                                                                                                                                                                                        
+     * Set a callback to be executed when an error occurs.                                                                                                                                                     
+     *                                                                                                                                                                                                         
+     * Subsequent calls to this method will overwrite the error handler. Setting                                                                                                                               
+     * the callback to null will clear the error handler.                                                                                                                                                      
+     *                                                                                                                                                                                                         
+     * @param callable|null $callback The callback to execute.                                                                                                                                                 
+     *                                                                                                                                                                                                         
+     * @return void                                                                                                                                                                                            
+     */                                                                                                                                                                                                        
+    public static function setErrorHandler(callable $callback = null)
```

Comment:
I think that should be `setErrorHandler(callable $callback)`. If you want to set none, that should be explicit by passing `null`.